compiler/go/compiled-with-go.yml

```diff
@@ -8,6 +8,6 @@ rule:
       - 49a34cfbeed733c24392c9217ef46bb6
   features:
     - or:
-      - string: /Go build ID:/
-      - string: /go.buildid/
-      - string: /Go buildinf:/
+      - string: "Go build ID:"
+      - string: "go.buildid"
+      - string: "Go buildinf:"
```

Comment:
```suggestion
      - string: /^Go build ID:/
      - string: /go\.buildid/
      - string: /^Go buildinf:/
```

Reply:
these need to be regexes so that they can match part of a string. typically the buildid string looks like `Go build ID: asdfafdsafdasfasd`